src/logging.js

```diff
@@ -1,57 +1,94 @@
 const winston = require('winston');
 const util = require('util');
+require("winston-daily-rotate-file");
+const chalk = require("chalk");
+let log = null;
 
-const Rotate = require('winston-logrotate').Rotate;
+const DEFAULT_LOG_LEVEL = "info";
+const TERM_COLORS = {
+  error: "red",
+  warn: "yellow",
+  info: "blue",
+  verbose: "white",
+  silly: "grey",
+};
 
-let log = null;
+function winstonFormatter (options) {
+  let level = options.level;
+  if(options.colorize && chalk.supportsColor) {
```

Comment:
I would suggest leaving not using colorize because that indicates that means we would be using winston's own colouring were we not overriding the format function.

I think I would have a parallel winstonColourFormatter or something that does what winstonFormatter does but with a coloured level. Then I'd factor out as much as possible from winstonFormatter to avoid duplication of code.